keyboards/xd84pro/keymaps/default/keymap.c

```diff
@@ -0,0 +1,36 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could we get a license header here?

Here's a sample one:

https://github.com/qmk/qmk_firmware/blob/4a83021fe0d07888e0e3bf161882c4ab7221a4a6/quantum/template/base/keymaps/default/keymap.c#L1-L15